library/oneview_version_facts.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2016-2017) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+ANSIBLE_METADATA = {'status': ['stableinterface'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: oneview_version
+short_description: Returns the range of possible API versions supported by the appliance.
+description:
+    - Provides an interface to return the range of possible API versions supported by the appliance.
+version_added: "2.3"
+requirements:
+    - "python >= 2.7.9"
```

Comment:
No need for this line

Reply:
can you please specify the line number which i need to remove from the documentation.

Reply:
[line 31](https://github.com/HewlettPackard/oneview-ansible/pull/301/files/ceae72f927c2934abefdb6fb48f74d37e1c842f5#diff-fe88f7dbd542d19b40ec1cdc5e599a86R31)